bundles/org.openhab.io.neeo/src/main/java/org/openhab/io/neeo/internal/NeeoApi.java

```diff
@@ -461,6 +469,41 @@ public NeeoDeviceKeys getDeviceKeys() {
         return deviceKeys;
     }
 
+    private synchronized HttpClient getHttpClient() {
+        int stackSize = httpClientStack.size();
+        if (stackSize == 0) {
+            int httpClientId = this.httpClientId + 1;
+            this.httpClientId = httpClientId;
+            String httpClientIdString = "neeo-" + brainId + "-" + httpClientId;
```

Comment:
To be checked if this naming is compliant with the OH convention.

Reply:
So this is a yes and a no.  The problem is, the brain ID is "NEEO-1234567" so we end up with neeo-NEEO.  Some of the bindings have a singular pool so it's just "OH-httpClient-unifi-520" or "OH-httpClient-remoteopenhab-1466" for example.  Those that have pools per thing seem to be like "OH-httpClient-plex-server-myPlex-81187" which are similar.  I don't mind changing it as long as we keep the brainId (or at least the hex string) to keep it unique.  Do you have a suggestion/recommendation to make it align better to the convention?

Reply:
When the client is attached to a particular thing, the recommendation is to use the method buildWebClientConsumerName from class ThingWebClientUtil.
But we can make an exception.